msi/nunit/engine-files.wxi

```diff
@@ -87,5 +88,34 @@
                 <File Id="nunit.agent.net40.addins" Source="$(var.InstallImage)nunit.agent.addins" />
             </Component>
         </ComponentGroup>
+        <ComponentGroup Id="NETCORE31_AGENT" Directory="NETCORE31_AGENT_DIR">
+            <Component Id="NUNIT_AGENT_NETCORE31" Location="local" Guid="21E39B4C-07E4-4C38-8F91-7030516127E2">
+                <File Id="nunit_agent_netcore31.exe"
+                      ProcessorArchitecture="msil"
+                      Source="$(var.InstallImage)bin/agents/netcoreapp3.1/nunit-agent.dll" />
+                <File Id="nunit_agent.runtimeconfig.json"
+                      Source="$(var.InstallImage)bin/agents/netcoreapp3.1/nunit-agent.runtimeconfig.json" />
```

Comment:
I'm not totally clear on what is and isn't required for .NET Core, but I _think_ this should also include nunit-agent.deps.json. 

I was also wondering about nunit-agent.dll.config, but I think is actually netfx specific, so it should probably be removed entirely from the netcoreapp agent? I'm not sure how to do that off the top of my head, will give it a go.

Reply:
I believe that `nunit-agent.deps.json` is only for use in building the project. It causes `nunit-agent.runtimeconfig.json` to be generated. If that's wrong, I can add it. Do you have different info? At this point all the tests pass using all the packages.

Reply:
Hmm, ok. I always thought it should be as it details where to load dependencies from at runtime. I'm struggling to find any formal documentation however, or any clear answer elsewhere. 

This page seems to be the closest there is. From my reading of that, for a framework-dependent deployment as ours is, the file should be included. What do you think?

https://github.com/dotnet/cli/blob/master/Documentation/specs/runtime-configuration-file.md